src/ui/czi-icon.css

```diff
@@ -68,3 +68,17 @@
   margin: 0 2px;
   vertical-align: super;
 }
+
+.czi-icon.subscript {
+  display: inline-block;
```

Comment:
To avoid duplication, merge common elements of this style with .czi-icon.superscript